api/api-data-sharing-settings-v1/src/main/java/com/thoughtworks/go/apiv1/datasharing/settings/DataSharingSettingsControllerV1Delegate.java

```diff
@@ -72,8 +75,10 @@ public void setupRoutes() {
 
             before("", mimeType, apiAuthenticationHelper::checkUserAnd403);
             before("", mimeType, this::checkAdminUserAnd403OnlyForPatch);
+            before("/notification", mimeType, apiAuthenticationHelper::checkUserAnd403);
             get("", this::getDataSharingSettings);
             patch("", mimeType, this::patchDataSharingSettings);
+            get("/notification", this::getDataSharingNotificationForCurrentUser);
```

Comment:
The endpoint reads as `/go/api/data_sharing/settings/notification` where it gives an impression of the API returning the settings notification.
Whereas, the API answers the authorization for showing the notification.
Can we say something like `/go/api/data_sharing/settings/notification_auth` ?

Reply:
ok